### PR TITLE
Cover Art API for Review

### PR DESCRIPTION
**Pull request by jy8474np**

### CODE REVIEW

**Comment by jy8474np** [comment]
Cover art .py file submitted for review.

File: cover_art_api.py [hunk 1]
```diff
@@ -1,35 +1,38 @@
...
 import requests
+import shutil
 import os
 import glob
 from pprint import pprint
 
 url = f"http://ws.audioscrobbler.com/2.0"
-#key = os.environ.get("COVER_KEY") # "2066002fabe29cf09d7ed5a00c804ac6"
+# key = os.environ.get("COVER_KEY") # "2066002fabe29cf09d7ed5a00c804ac6"
 key = "2066002fabe29cf09d7ed5a00c804ac6"
```
Comment (yp0975eu) [inline cover_art_api.py:11]: We will need to get this key removed.

**Review by yp0975eu — APPROVED** [review]

File: cover_art_api.py (hunk 1 above)
Comment (jy8474np) [reply cover_art_api.py:11]: Yes - That's still an issue. Thanks for reminding! I know you two got yours to work without issue so let's review together.

**Review by ashley7295 — APPROVED** [review]
Just need to adjust the key's otherwise looks good from our in person zoom code review

**Review by Issemoh — APPROVED** [review]

**Pull request merged**